### PR TITLE
This is the updated file of Q&As

### PR DESCRIPTION
**Pull request by vagelisx**

### CODE REVIEW

**Comment by patric94** [comment]
Good job @vagelisx !
If you have the time, could you please do the following changes?
- Add any questions added after your PR
- Please replace `Question:` with `Q:` and `Answer:` with `A:`
- Please add after every Q&A a horrizontal line (three '-') to increase readability of document
Thanks a lo!

**Comment by kostasx** [comment]
@patric94 @vagelisx I've updated the requirements and we can either use **Q: A:** or **Question: Answer:** or even **QUESTION: ANSWER:** for separating the Q&A. So, there's no need to change the **Question: Answer:**. The parser will take care of the lowercase/uppercase and various formats. :)

**Comment by vagelisx** [comment]
> Good job @vagelisx ! If you have the time, could you please do the following changes?
> 
>     * Add any questions added after your PR
> 
>     * Please replace `Question:` with `Q:` and `Answer:` with `A:`
> 
>     * Please add after every Q&A a horrizontal line (three '-') to increase readability of document
>       Thanks a lo!

@patric94 cc @kostax : I made the changes anyway to practice the git commands and processes. I will keep my eye on the Q&A's next week, you can continue working on more demandin issues.

**Comment by kostasx** [comment]
@vagelisx Please go through the [git diff](https://github.com/in-tech-gration/WDX-180/pull/17/files) carefully and fix the content that contains wrong replacements of Q. 

As I mentioned, it's not necessary from now on to stick to Q/A. Question/Answer or QUESTION/ANSWER will do just fine.

Don't forget to add "---" between each question:

```markdown
Q: Why I am here?
A: Don't know

---

Q: What is the meaning of it all?
A: 42
```

File: staff/archive/slido.md [hunk 1]
```diff
@@ -183,7 +323,7 @@
 
   ---
 
-  Q: I have a question regarding VS Code. I don't see any day 04 exercises in my curriculum or user. How do I complete the assignment? The only thing I see under day 04 is README.md
+  Q: I have a Q regarding VS Code. I don't see any day 04 exercises in my curriculum or user. How do I complete the assignment? The only thing I see under day 04 is README.md
```
Comment (kostasx) [inline staff/archive/slido.md:326]: "Q: I have a Q regarding VS Code." @vagelisx

**Review by kostasx — CHANGES_REQUESTED** [review]

File: staff/archive/slido.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ A: There are certain limitations though, when you are part of the free plan on G
 
  Q: touch command doesnt work on command line
 
- A: This Q was covered extensively on the Q&A section: 
+ A: This Q was covered extensively on the Q&A live section: https://www.youtube.com/watch?v=mhUiYaZjEvU&list=PLdo7hJB0agEkkLFrssQCGWnV08Q50X_M0&index=5
```
Comment (kostasx) [inline staff/archive/slido.md:73]: Another one here. @vagelisx

**Review by kostasx — CHANGES_REQUESTED** [review]

**Comment by kostasx** [comment]
Thank you @vagelisx.

**Pull request closed without merging**